openjdk/playlist.xml

```diff
@@ -2635,6 +2635,35 @@
 			<feature>FIPS140_3_OpenJCEPlusFIPS.FIPS140-3:nonapplicable</feature>
 		</features>
 	</test>
+	<test>
+		<testCaseName>jdk11_tier1_cipher</testCaseName>
```

Comment:
This test runs for JDK11+. Please remove `jdk11` from testCaseName. To align with other test name, we should  call it `jdk_cipher`.

Reply:
I used this name to be consistent with related targets `jdk11_tier1_buffer`, `jdk11_tier1_iso8859`, `jdk11_tier1_pack200` (above new one in playlist).  Only pack200 is 11 other ones are 11+. (Non of these have name in `TEST.groups` in openjdk.)